h/static/scripts/annotator/plugin/test/pdf-metadata-test.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+var PDFMetadataService = require('../pdf-metadata');
+
+describe('pdf-metadata', function () {
+  var pdfMetadata;
+  var fakePDFViewerApplication = {
+    'documentFingerprint': 'fakeFingerprint',
+    'documentInfo': {
+      'Title': 'fakeTitle',
+    },
+    'metadata': {
+      'metadata': {
+        'dc:title': 'fakeTitle',
+      }
+    },
+    'url': 'fakeUrl',
+  };
+  var sandbox;
+
+  beforeEach(function () {
+    pdfMetadata = new PDFMetadataService(fakePDFViewerApplication);
+    pdfMetadata._documentLoaded = Promise.resolve();
+    sandbox = sinon.sandbox.create();
+  });
+
+  afterEach(function () {
+    sandbox.restore();
+  });
+
+  describe('.getUrn', function () {
```

Comment:
Nit. Should use `#name` (note leading '#') to refer to instance fields/methods in describe/it tests, following JSDoc conventions. Using a period denotes a class property.